package.json

```diff
@@ -57,6 +62,16 @@
     "tslint-config-prettier": "^1.18.0",
     "typescript": "^4.1.2"
   },
+  "jest": {
+    "coverageThreshold": {
+      "global": {
+        "branches": 85,
+        "functions": 85,
+        "lines": 85,
```

Comment:
Could we focus on 95+ from the beginning itself?

Reply:
yes sure I will do that